lib/truffle/truffle/cext.rb

```diff
@@ -843,7 +843,7 @@ def rb_funcall_with_block(recv, meth, args, block)
   end
 
   def rb_respond_to(object, name)
-    object.respond_to?(name)
+    rb_obj_respond_to(object, name, false);
```

Comment:
`false => 0` here (it's 0 for rb_obj_respond_to because it comes from C as an `int`).
Or we could just inline the logic since it's so simple.

Reply:
I'll fix it.

Reply:
Ah, for sure! We have `priv != 0` in `rb_obj_respond_to`

Reply:
It is curious that not a single test failed .. maybe if start everything (not just fast), it will be red

Reply:
There are not many specs for those 2 C API functions: https://github.com/ruby/spec/blob/01d0d9940f1b393eaf92440efe9adf1ebcd29374/optional/capi/object_spec.rb#L108-L127

Would be good to add more and actually check they call `respond_to?` on the receiver.

Reply:
Are you suggesting adding soecs as part of the current PR or doing it as a next step?

Reply:
In this case I'd suggest as a next step since the PR is already running in CI.

Reply:
Got it, I've added it to my personal backlog 😎